code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -165,6 +165,9 @@
 /obj/item/weapon/gun/ballistic/shotgun/boltaction/enchanted/arcane_barrage/discard_gun(mob/user)
 	return
 
+/obj/item/weapon/gun/ballistic/shotgun/boltaction/attack_self()
```

Comment:
wizards don't need to unload a boltaction enchanted rifle either as I believe that breaks it too.